src/assets/scss/_tabs.scss

```diff
@@ -146,36 +150,42 @@
 
   &.nav-tabs-primary {
     > li::after {
+      /* stylelint-disable-next-line */
       background-color: #6d5cae;
     }
   }
 
   &.nav-tabs-success {
     > li::after {
+      /* stylelint-disable-next-line */
       background-color: #10cfbd;
     }
   }
 
   &.nav-tabs-complete {
     > li::after {
+      /* stylelint-disable-next-line */
       background-color: #48b0f7;
     }
   }
 
   &.nav-tabs-danger {
     > li::after {
+      /* stylelint-disable-next-line */
       background-color: #f55753;
     }
   }
 
   &.nav-tabs-warning {
-    > li::after {
+    > li::after 
+    /* stylelint-disable-next-line */{
```

Comment:
Is this behaving properly? Comparing this line to #173/#174, the curly bracket's in a different place.

Reply:
That's my bad.